https-server/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEowIBAAKCAQEAmfm2cOU49Ptbn3HKNY6/WPpt05Lumt43R863mn8QpMg5MeEw
+dn0pWb83yOyFc+h40ZpEnCO8+HgslABhlAL0W8WFiUaFpH2IX5orPn5N6tDd7Nc+
```

Comment:
Uh do we want the private key for this committed into git?? 🤔

Reply:
This isnt a real private key. Its a fake one, self signed one, that is used for running a https server locally (because you can't access the webcam API if its not https)